src/widgets/texturepickerwidget.cpp

```diff
@@ -59,7 +59,7 @@ QString TexturePickerWidget::loadTexture()
 {
     QString dir = QApplication::applicationDirPath();
     return QFileDialog::getOpenFileName(this, "Open Texture File",
-                                        dir, "Image Files (*.png *.jpg *.bmp)");
+                                        "", "Image Files (*.png *.jpg *.bmp)");
```

Comment:
Funny how this merits a whole commit and at the same time you cram 30 things into one.